README.md

```diff
@@ -12,27 +12,58 @@ Inspired by the simplicity of [discord.py](https://github.com/Rapptz/discord.py)
 
 ```cpp
 #include <iostream>
+#include <vector>
+#include <algorithm>
+#include <iterator>
 #include <dpp.h>
 
-int main()
+std::vector<std::string> owners = { "ID1", "ID2" };
+
+void main()
 {
- const std::string token = "Bot token here";
- dpp::Client client;
-
- client.onReady = [&]() {
-  std::cout << "hello, c++!\n";
-  std::cout << client.user.id << "\n\n";
- };
-
- client.onMessage = [&](dpp::Message message) {
-  if (message.content == "jaaj?") {
-   message.channel.send("jooj"); // or message.reply("jooj"), which mentions the message author
-  }
- };
- 
- client.run(token);
-}
+    const std::vector<std::string> owners = { "ID1", "ID2", "..." }; //You can add some IDs(Bot Owners only) !
+    const std::string prefix = ">"; //Bot Prefix
+    
+    const std::string token = "Bot token";
+
+    dpp::Client client;
+
+    client.onReady = [&]() {
+        std::cout << client.user.name << " ready!\n\n";
+    };
+
+
+    client.onMessage = [&](dpp::Message message) {
+
+        if (message.author.id == client.user.id) return;
+        //if (message.author.bot) return message.channel.send("Bots aren't allowed"); //message.author.bot doesn't return user for the moment :/
+        //Waiting the `startWith` to add return if the prefix isn't in the message and more..
+        if (message.content == prefix + "ownersonly") { //Prefix with command
+            if (std::find(std::begin(owners), std::end(owners), message.author.id) != std::end(owners)) { //Little example for owners only(will be updated later)
+                message.reply("Allowed");
+                return;
+            }
+            else {
+                message.reply("Not Allowed");
+                return;
+            };
+
+        }
+        else if (message.content == prefix + "help") { //Prefix with command
+            message.reply("In progress..");
+        }
+        else {
+            //message.reply("command"); //Need to wait for startWith to avoid a response without prefix!
+        };
+    };
+    client.run(token);
+};
+
```

Comment:
Although I like what you went for here, I think we should keep the REAME.md example as simple as we can, and includen this in some documentation later.

Reply:
I leave it like that for those who want some more advanced examples but you can modify it if you want to

Reply:
I'll add such examples in the documentation website